### ISSUE TITLE
✨ Add current location interface in search input

### ISSUE THREAD

**Issue by HarshInvader** [IndiaVaccineIN/india-vaccine-frontend#11]
API only allows search by pincode and district so have to find a way to map current location data to pincode or district.

**Comment by umanghome** [comment]
My location would technically fall under both Bangalore Urban and BBMP. :/

**Comment by umanghome** [comment]
And pincode search does not work most of the time. Hasn't worked for me yet.

**Comment by HarshInvader** [comment]
Thanks for your points @umanghome. We will have to figure out a good way to map a location to nearby pincodes and then show the centre.

**Comment by AniketSK** [comment]
@umanghome the cowin API, confusingly isn't about:
"Give me vaccination centers close/closest to this pin"
It's like:
"Show me all vaccination centres that have this exact pin code".

@HarshInvader ~~how about an api that asks for people's locations via the browser and then compares that against the lat/lon defined in the api results of centers?~~ Sorry I missed the call yesterday but I could take this if there's nothing else too pressing?
Looking up their coordinates,  I can see they're rounded off to the point of uselessness.
How about getting distances via pin codes and showing people all the ones around them?

**Comment by HarshInvader** [comment]
Yes @AniketSK it might be handy to connect with someone from the back-end and data team for this as well to understand the state of data. You can ping Joel on Discord
